src/utils/consts.js

```diff
@@ -22,6 +22,7 @@ const consts = {
   concernArray: ["No Concern", "Some Concern", "High Concern"],
   locationArray: ["Facility 1", "Facility 2", "Placed"],
   roleArray: ["Service", "Companion"],
+  userRoleArray: ["Admin", "Instructor", "Volunteer", "Inactive"],
```

Comment:
leadership decided to change this to 
`["Admin", "Instructor", "Volunteer/Recipient", "Inactive"]`

Reply:
👍 Done